runtimes/spiritnet/src/lib.rs

```diff
@@ -913,12 +916,12 @@ construct_runtime! {
 		TransactionPayment: pallet_transaction_payment::{Pallet, Storage, Event<T>} = 7,
 
 		// Consensus support.
-		// The following order MUST NOT be changed: Authorship -> Staking -> Session -> Aura -> AuraExt
-		Authorship: pallet_authorship::{Pallet, Call, Storage} = 20,
-		ParachainStaking: parachain_staking = 21,
-		Session: pallet_session = 22,
-		Aura: pallet_aura = 23,
+		// The following order MUST NOT be changed: AuraExt -> Aura -> Session -> Staking -> Authorship
 		AuraExt: cumulus_pallet_aura_ext = 24,
+		Aura: pallet_aura = 23,
+		Session: pallet_session = 22,
+		ParachainStaking: parachain_staking = 21,
+		Authorship: pallet_authorship::{Pallet, Call, Storage} = 20,
```

Comment:
Weirdly, when running benchmarks, an error is thrown with this config under `AllPalletsWithSystem`:
```Thread 'main' panicked at 'AuRa authorities empty, maybe wrong order in `construct_runtime!`?', /home/willi/.cargo/git/checkouts/cumulus-59522f43471fa161/35dd14f/pallets/aura-ext/src/lib.rs:96```

When reverting the order (:= `REV` = Authorship -> Staking -> Session -> Aura -> AuraExt), the benchmarks work. That does not make sense though, as we would then run everything in reverse order compared to our current execution. Moreover, there was a user on Discord having problems with `REV` and `AllPalletsWithSystem`. The issue there was that session keys were not propagated in `--dev` environment.

Need to investigate locally...

Reply:
After intensive testing, I can say that Aura -> Session -> Staking -> Authorship is fine. However, AuraExt needs to be after Aura because the latter sets the authorities which are queried by AuraExt. Should hopefully be fixed now, see [134d428](https://github.com/KILTprotocol/kilt-node/pull/416/commits/134d428491556358de6b6d271bcb649590ac75e8)